### PR TITLE
Vscode dependency update

### PR DESCRIPTION
**Pull request by hunger**

### CODE REVIEW

File: editors/vscode/src/wasm_preview.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // Copyright © SixtyFPS GmbH <info@slint.dev>
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
 
+// cSpell: ignore winit
```
Comment (ogoffart) [inline editors/vscode/src/wasm_preview.ts:4]: unrelated, this should probably be ignored globally as this is such a common term in slint

**Review by ogoffart — APPROVED** [review]

**Comment by tronical** [comment]
> I am fine with keeping this till after release though.

We tend to do more QA before the release than after. I’d say don’t wait.

**Comment by tronical** [comment]
I suspect that this is what causes the nightly to fail https://github.com/slint-ui/slint/actions/runs/6515431431/job/17697975352

**Comment by ogoffart** [comment]
I pushed https://github.com/slint-ui/slint/commit/ef39adce30d33b86634e5359db03a18d521a9bc8 as a prospective fix.
This means that the extension will need vscode 1.82 which is the August 2023   (latest is vscode 1.83 from Sept)
I wonder if people use old version of vscode so much. It's probably fine to require a recent version.

**Comment by tronical** [comment]
Thanks! (Agreed about the min version)

File: editors/vscode/src/common.ts [hunk 2]
```diff
@@ -132,21 +134,43 @@ export function languageClientOptions(
...
     const properties_provider = new PropertiesViewProvider(
         context.extensionUri,
     );
 
+    context.subscriptions.push(statusBar);
+    statusBar.text = "Slint";
+
+    client.add_updater((cl) => {
+        if (cl !== null) {
+            cl.registerFeature(new snippets.SnippetTextEditFeature());
```
Comment (jpnurmi) [inline editors/vscode/src/common.ts:152]: Moving the snippet feature registration here caused the "Wrap in element" and "Remove element" code actions to go missing. I think the static feature needs to be registered before starting the client. Would it be ok to move this either back to `browser.ts` and `extension.ts` or add some other helper here that gets called before starting the client?

**Pull request merged**